jenkins/internal-ci/centos-7.9.2009/pr/component_test/pyutils-premerge.groovy

```diff
@@ -0,0 +1,289 @@
+                            parameters: [
+                                string(name: 'CORTX_RE_REPO', value: "https://github.com/Seagate/cortx-re/"),
+                                string(name: 'CORTX_RE_BRANCH', value: "main"),
+                                string(name: 'CORTX_IMAGE', value: "${env.cortx_all_image}"),
+                                string(name: 'hosts', value: "${singlenode_host}")
+                            ] 
+                        }
+                    }
+                }
+         	    stage ("Deploy 3Node") {
+                    when { expression { params.DEPLOY_BUILD_ON_NODES ==~ /Both|3node/ } }
+                    steps {
+                        script { build_stage = env.STAGE_NAME }
+                        script {
+                            build job: '/Cortx-PR-Build/Cortx-Deployment/Kubernetes/K8s-3N-deployment', wait: true,
+                            parameters: [
+                                string(name: 'CORTX_RE_BRANCH', value: "main"),
+                                string(name: 'CORTX_RE_REPO', value: "https://github.com/Seagate/cortx-re/"),
+                                string(name: 'CORTX_IMAGE', value: "${env.cortx_all_image}"),
+                                text(name: 'hosts', value: "${threenode_hosts}"),
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+
+        stage ("Test RPM") {
+            steps {
+                script {
+                    sh label: 'RUN command to test rpm', script: '''
+                        if [ $DEPLOY_BUILD_ON_NODES == '1node' ]; then
+                            echo $singlenode_host | sed 's/,/ /g' | sed 's/=/ /g' |  awk -F ' ' '{ print $2" "$4" "$6 }' > server_info.txt
+                            NODE_HOST=$( cat server_info.txt | awk -F ' ' '{ print $1 }')
+                            NODE_USER=$( cat server_info.txt | awk -F ' ' '{ print $2 }')
+                            NODE_PASS=$( cat server_info.txt | awk -F ' ' '{ print $3 }')
+                        else
+                            echo $threenode_hosts | head -n1 | sed 's/,/ /g' | sed 's/=/ /g' |  awk -F ' ' '{ print $2" "$4" "$6 }' > server_info.txt
+                            NODE_HOST=$( cat server_info.txt | awk -F ' ' '{ print $1 }')
+                            NODE_USER=$( cat server_info.txt | awk -F ' ' '{ print $2 }')
+                            NODE_PASS=$( cat server_info.txt | awk -F ' ' '{ print $3 }')
+                        fi
+                        yum install sshpass -y
+                        sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} kubectl exec '$(kubectl get pods | grep "cortx-data" | awk '{print$1}')' --container cortx-hax --  yum install http://cortx-storage.colo.seagate.com/releases/cortx/github/pr-build/${BRANCH}/${COMPONENT_NAME}/${BUILD_NUMBER}/cortx_iso/cortx-py-utils-test-1.0.0-1.noarch.rpm -y
```

Comment:
Please check if RPM version will be always like `cortx-py-utils-test-1.0.0-1.noarch.rpm`